verifyCredentials.js

```diff
@@ -1,21 +1,16 @@
-var sftp = require("./lib/sftp.js");
-
-module.exports = verify;
+const sftp = require('./lib/sftp.js');
 
 function verify(cfg, cb) {
```

Comment:
Better to leave some logs here

Reply:
Just in `verifyCredentials` or throughout?